builtin/game/item_entity.lua

```diff
@@ -172,13 +175,15 @@ core.register_entity(":__builtin:item", {
 		-- If node is not registered or node is walkably solid and resting on nodebox
 		local v = self.object:getvelocity()
 		if not core.registered_nodes[nn] or core.registered_nodes[nn].walkable and v.y == 0 then
-			if self.physical_state then
+			if self.physical_state
```

Comment:
Am I seeing this correct that only items with `physical_state` set to `true` can merge with items with it set to `false`? Is this okay this way?